manifests/gateways/istio-ingress/values.yaml

```diff
@@ -48,9 +48,6 @@ gateways:
     - port: 15032
       targetPort: 15032
       name: tracing
-    - port: 31400
-      targetPort: 31400
-      name: tcp
```

Comment:
is this port used for anything @howardjohn ? it's missing in default.yaml so i took it out but ingress failing is making me think it could be used for tests.

Reply:
I think its use for some doc? If i recall correctly it was just used in demo.yaml though

Reply:
I don't think this is why the ingress test fails though

Reply:
31400 is used, as @howardjohn has indicated, in the docs. I think we need to find a better solution in the docs then using port 31400 and always exporting this port via ingress. If this port is removed from ingress though, the docs will no longer work in many tasks. We test for this in the test cases:

See https://github.com/istio/istio.io/issues/6631#issuecomment-592096687

Cheers
-steve